uber/templates/emails/reg_workflow/hotel_booking_info.html

```diff
@@ -0,0 +1,22 @@
+<html>
+<head></head>
+<body>
+
+<p>{{ attendee.first_name }},</p>
+
+<p>
+You've expressed interest in booking a hotel room for {{ c.EVENT_NAME }} this
+coming {{ event_dates() }}. Follow the link below to book a room before they
+sell out:
+</p>
```

Comment:
The copy in this file needs some TLC. Probably need to get input from reg, hotels, @nickthenewbie1, etc...